sql/moz-fx-data-shared-prod/zoom/meeting_participants/metadata.yaml

```diff
@@ -0,0 +1,5 @@
+friendly_name: Zoom meeting participants
+description: |-
+  Mozilla's Zoom meeting participants as synced by Fivetran starting in January 2025.
+labels:
+  authorized: true
```

Comment:
@whd can you confirm these authorized views will be automatically authorized when they're deployed?